helm/wind-river-cloud-platform-deployment-manager/templates/crds.yaml

```diff
@@ -578,6 +578,9 @@ spec:
                 description: MaxCPUMhzConfigured defines the maximum limit of the CPU mhz configured on the host.
                 pattern: ^[1-9][0-9]*$
                 type: string
+              appArmor:
+                description: AppArmor defines the security model of the host
+                type: string
```

Comment:
I suggest the type be boolean. As this this CR is configured as "enabled/disabled"

Reply:
System CLI / REST API have apparmor as string. So leaving it as string